### PR TITLE
[release-2.4.0 < T0976-MG] Add command to return startup config

### PR DESCRIPTION
**Pull request by gvolfing**
### Description

Modify Changelog by adding the new SHOW CONFIG command and updating the Configuration Settings page with the command.

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [x] Other (please describe): Changelog.

Closes (https://github.com/memgraph/memgraph/issues/397)
[Memgraph PR](https://github.com/memgraph/memgraph/pull/459)

### Checklist:

- [x] I checked all content with Grammarly
- [x] I performed a self-review of my code
- [x] I made corresponding changes to the rest of the documentation
- [x] The build passes locally
- [x] My changes generate no new warnings or errors

### CODE REVIEW

**Comment by vpavicic** [comment]
We can also add this info here:
https://memgraph.com/docs/memgraph/how-to-guides/config-logs

@gvolfing do you mind if I do a commit on this PR?

**Review by antaljanosbenjamin — CHANGES_REQUESTED** [review]
Everything you wrote seems okay, but I would also explicitly mention the `CONFIG` privilege is required to run the query. Maybe the best place is the `configuration.md` file in the reference guide.

Furthermore in `docs/how-to-guides/manage-user-privileges.md` the description of `CONFIG` privilege is about configuring Memgraph runtime. Does that include getting the configuration also? @vpavicic what do you think?

**Comment by vpavicic** [comment]
> 

I think it should be explicitly stated that it includes getting the config as well :)

**Comment by gvolfing** [comment]
@vpavicic @antaljanosbenjamin 
Made it clear in `docs/how-to-guides/manage-user-privileges.md` that the `CONFIG` privilege level is required for the query and also mentioned it in `configuration.md `.

**Review by vpavicic — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ sidebar_label: Changelog
 - Query `REGISTER REPLICA replica_name SYNC` no longer supports `TIMEOUT` parameter. To mimic the previous behavior of `REGISTER REPLICA replica_name SYNC WITH TIMEOUT 1`, one should use `REGISTER REPLICA replica_name ASYNC` instead. [#423](https://github.com/memgraph/memgraph/pull/423)
 - Make behavior more [openCypher](http://opencypher.org/) compliant regarding checking against `NULL` values is `CASE` expressions. [#432](https://github.com/memgraph/memgraph/pull/432)
 - Previously registered replicas are automatically registered on restart of Memgraph. [#415](https://github.com/memgraph/memgraph/pull/415)
+- Add new command `SHOW CONFIG` that returns the configuration of the currently running Memgraph instance. [#459](https://github.com/memgraph/memgraph/pull/459)
```
Comment (kostasrim) [inline docs/changelog.md:32]:
```suggestion
- Added query `SHOW CONFIG` that returns the configuration of the Memgraph instance. [#459](https://github.com/memgraph/memgraph/pull/459)
```

File: docs/how-to-guides/manage-user-privileges.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ We could naturally cluster those privileges into groups:
   * Privilege to try freeing memory (`FREE_MEMORY`)
   * Privilege to use trigger queries (`TRIGGER`)
   * Privilege to use stream queries (`STREAM`)
-  * Privilege to configure Memgraph during runtime (`CONFIG`)
+  * Privilege to configure Memgraph during runtime and to attain the configuration of the given Memgraph instance(`CONFIG`)
```
Comment (kostasrim) [inline docs/how-to-guides/manage-user-privileges.md:77]:
```suggestion
  * Privilege to configure Memgraph during runtime and to access the configuration of the given Memgraph instance(`CONFIG`)
```

File: docs/reference-guide/configuration.md [hunk 3]
```diff
@@ -14,6 +14,11 @@ variable or by passing arguments on the command line.
 
 Each configuration setting is in the form: `--setting-name=value` .
 
+Privilege level `CONFIG` is required to be able to fetch the configuration and it can be attained by calling the following query:
```
Comment (kostasrim) [inline docs/reference-guide/configuration.md:17]:
```suggestion
Privilege level `CONFIG` is required to be able to fetch the configuration via the following query:
```

**Review by kostasrim — APPROVED** [review]

File: docs/how-to-guides/manage-user-privileges.md (hunk 2 above)
Comment (vpavicic) [reply docs/how-to-guides/manage-user-privileges.md:77]: to access configuration to me implies I can do something with it (like change it) :)

**Pull request merged**